.github/workflows/ci.yml

```diff
@@ -142,9 +142,16 @@ jobs:
         id: electron-unit-tests
         run: DISPLAY=:10 ./scripts/test.sh --runGlob "**/sql/**/*.test.js" --coverage
 
-      - name: Run Extension Unit Tests # {{SQL CARBON EDIT}} Rename to core for clarity
-        id: electron-extension-unit-tests
+      - name: Run Extension Unit Tests (Continue on Error) # {{SQL CARBON EDIT}} Rename to core for clarity
```

Comment:
the comments here are copied from previous lines, please update it as well as the comment for step  `Run Extension Unit Tests`